src/libraries/Native/Unix/System.Globalization.Native/pal_collation.c

```diff
@@ -689,13 +689,14 @@ static int32_t GetCollationElementMask(UColAttributeValue strength)
     }
 }
 
-static int32_t inline SimpleAffix_Iterators(UCollationElements* pPatternIterator, UCollationElements* pSourceIterator, UColAttributeValue strength, int32_t forwardSearch)
+static int32_t inline SimpleAffix_Iterators(UCollationElements* pPatternIterator, UCollationElements* pSourceIterator, UColAttributeValue strength, int32_t forwardSearch, int32_t captureOffset, int32_t* pCapturedOffset)
```

Comment:
>captureOffset [](start = 179, length = 13)

can we get rid of captureOffset and check only pCapturedOffset != null?

Reply:
I am seeing the callingsite which I believe can easily updated and pass null when needed only.

---
In reply to: [470110920](https://github.com/dotnet/runtime/pull/40752#discussion_r470110920) [](ancestors = 470110920)